code/modules/mob/inventory/_docs.dm

```diff
@@ -57,6 +57,53 @@ Ovverided for human proc check "slot_back" first.
 
 
 
+
+// UNEQUIP //
+
+/mob/proc/u_equip(obj/item/Item)
+/*
+This is an UNSAFE proc.
+Inequip analogue for equip_to_slot(...)
+
+It merely handles the actual job of equipping. Drop vars, update icons. Also calls Item.dropped(mob, slot).
+Does nothing else.
+
+DO NOT CALL THIS PROC DIRECTLY. It is meant to be called only by other inventory procs.
+It's probably okay to use it if you are transferring the item between slots on the same mob,
+but chances are you're safer calling remove_from_mob() or drop_from_inventory() anyways.
+
+It calls:
+	- Item.dropped(Mob, slot)
+*/
+
+
+/mob/proc/drop_from_inventory(obj/item/Item, atom/Target, drop_flag)
+/*
+This is a SAFE proc. You can use this instead of u_equip()!
+Remove an Item from a mob and forceMove it into Target atom
+Remove from mob means:
+	- remove from mob client screen (if client exist)
+	- remove from mobs contents (Item.forceMove(...)) / inventory (mob.u_equip(...))
+
+If no Target is specify item will be moved to mob.loc and forceMoved with MOVED_DROP flag.
+Check /atom/movable/proc/forceMove(...) for more info about flags.
+
+It calls:
+	- Mob.u_equip(Item)
+	- Item.forceMove(Target)
+*/
+
+
+/mob/proc/unEquip(obj/item/Item, atom/Target)
+/*
+This differs from drop_from_inventory(...) in that it checks if the item can be unequipped first.
```

Comment:
IF only this one checks, then how can drop_from_inventory be a safe proc?

And if this is the one that checks, and u_equip is the one that does the heavy lifting, why do we need the middleman of drop_from_inventory at all?